### ISSUE TITLE
Instructions for Astroplant are like a scavenger hunt

### ISSUE THREAD

**Issue by jklingen92** [AstroPlant/Casing#9]
The instructions for building this thing are scattered all over the place. 

The kit points me to the github, (not the casing repository), no repository that I've seen points to the gitbook. There needs to be a single source for all resources that points everywhere you'd need to go and what order you should go there in.

**Comment by thiemehennis** [comment]
You're correct! We've got a team.working on Frontend now and precisely what you are suggesting. Sorry for the mess, we hope to close this issue soon.

**Comment by thiemehennis** [comment]
@danielsteginga @GernotAP

**Comment by danielsteginga** [comment]
We hope to do an update of our website and instructions in July
